core/templates/pages/topic-editor-page/navbar/topic-editor-navbar.component.ts

```diff
@@ -220,10 +220,26 @@ export class TopicEditorNavbarComponent implements OnInit,
     return validationIssuesCount + prepublishValidationIssuesCount;
   }
 
+  getMobileNavigatorText(): string {
+    const activeTab = this.getActiveTabName();
+    if (activeTab === 'main') {
+      return 'Editor';
+    } else if (activeTab === 'subtopic_editor') {
+      return 'Editor';
+    } else if (activeTab === 'subtopic_preview') {
+      return 'Preview';
+    } else if (activeTab === 'questions') {
+      return 'Questions';
+    } else if (activeTab === 'topic_preview') {
+      return 'Preview';
+    }
+  }
+
   openTopicViewer(): void {
     this.showNavigationOptions = false;
     let activeTab = this.topicEditorRoutingService.getActiveTabName();
-    if (activeTab !== 'subtopic_editor') {
+    if (
+      !(activeTab === 'subtopic_editor' || activeTab === 'subtopic_preview')) {
```

Comment:
I think this can be simplified:
```
~(a or b) = ~a and ~b
```
```suggestion
    if (activeTab !== 'subtopic_editor' & activeTab !== 'subtopic_preview') {
```

Reply:
Thanks @Lawful2002 for you review.
I made the suggested changes. And below it the screen recording showing, how I navigated to topic editor page
[1.webm](https://github.com/oppia/oppia/assets/92575005/f2f0b8ed-b4c2-45d5-b24d-26dae190d0ce)
Thank you